src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
don't use this selector for zeroing your indents

Reply:
what should I use instead?